Project.toml

```
@@ -21,7 +21,7 @@ UUIDs = "cf7118a7-6976-5b1a-9a39-7adc72f591a4"
 [compat]
 Colors = "0.10, 0.11, 0.12"
 ContextVariablesX = "0.1"
-MemPool = "0.3.5"
+MemPool = "0.4.1"
```

```suggestion
MemPool = "0.3.5, 0.4"
```

We should be able to support both with this change (the kwarg was dead anyway).